frontend/src/lib/actionUtils.ts

```diff
@@ -3,8 +3,10 @@ import { cssEscape } from 'lib/utils/cssEscape'
 import { ElementType } from '~/types'
 
 // these plus any element with cursor:pointer will be click targets
-export const CLICK_TARGETS = ['a', 'button', 'input', 'select', 'textarea', 'label', 'h1']
+export const CLICK_TARGETS = ['a', 'button', 'input', 'select', 'textarea']
+export const EXPERIMENT_TARGETS = ['label', 'h1', 'h2', 'h3', 'h4', 'span', 'img']
```

Comment:
@jurajmajerik or @corywatilo is this a good set of elements to run experiments on? I don't want to put in a `*` here and would rather limit the set.

Reply:
I think `div` and `p` should be here as well? We can always extend this easily..

Reply:
Tbh I think this is a huge limitation if we're deliberately specifying a select amount of tags. Going to lead to a lot of user frustration if they can't just target exactly what they're looking for.

Reply:
oh, I've just realized there are semantic tags too..